src/components/auth/Login.jsx

```diff
@@ -48,13 +48,19 @@ const Login = () => {
     }
   }, [isLoggedIn]);
 
+  const handleForgotPassword = () => {
```

Comment:
This could have been placed in one line 
const handleForgotPassword = () => navigate("/forgot-password");

Reply:
I do not have it is a big deal until we have some document with special rules. Both functions are valid and commonly used.

Reply:
Agreed.  I would generally recommend one-liners when you have multiple functions that have a single expression/statement and are stacked. So reducing their content would make reading them easier against each other. But more importantly follow whatever pattern your codebase has established.